search/search_quality_tests/search_quality_tests.cpp

```diff
@@ -122,11 +150,18 @@ int main(int argc, char * argv[])
     for (auto & map : mwms)
       map.SyncWithDisk();
   }
-  for (auto const & mwm : mwms)
+  cout << "Mwms used in all search invocations:" << endl;
+  for (auto & mwm : mwms)
   {
-    LOG(LINFO, ("Registering map:", mwm.GetCountryName()));
+    mwm.SyncWithDisk();
+    cout << mwm.GetCountryName() << " " << ReadVersionFromHeader(mwm) << endl;
     engine.RegisterMap(mwm);
   }
+  cout << endl;
+
+  // auto const & viewport = kDefaultViewport;
```

Comment:
How about to add a command line flag for default viewports, like Moscow, London, Zurich, Munich?